bulk-load/src/test/resources/publication.json

```diff
@@ -0,0 +1,134 @@
+{
+  "type": "Publication",
+  "id": "https://example.org/publication/__IDENTIFIER__",
+  "identifier": "__IDENTIFIER__",
+  "status": "Draft",
+  "owner": "eier@example.org",
+  "publisher": {
+    "type": "Organization",
+    "id": "http://example.org/org/123",
+    "labels": {
+      "no": "Eksempelforlaget"
+    }
+  },
+  "createdDate": "2020-03-30T05:51:02.487Z",
+  "modifiedDate": "2020-03-30T05:51:02.487Z",
+  "handle": "http://example.org/handle/123",
+  "link": "http://example.org/link",
+  "entityDescription": {
+    "type": "EntityDescription",
+    "publicationType": "JournalContent",
+    "mainTitle": "Hovedtittelen",
+    "alternativeTitles": {
+      "en": "English title"
+    },
+    "reference": {
+      "type": "Reference",
+      "doi": "https://doi.org/10.1016/j.foodres.2022.111012",
+      "publicationContext": {
+        "type": "Journal"
+      },
+      "publicationInstance": {
+        "type": "AcademicArticle"
+      }
+    },
+    "language": "http://example.org/norsk",
+    "date": {
+      "type": "PublicationDate",
+      "year": "2020",
+      "month": "4",
+      "day": "7"
+    },
+    "contributors": [
+      {
+        "type": "Contributor",
+        "identity": {
+          "type": "Identity",
+          "id": "http://example.org/person/123",
+          "name": "Navnesen, Navn",
+          "nameType": "Personal",
+          "orcId": "orc123",
+          "arpId": "arp123"
+        },
+        "affiliations": [
+          {
+            "type": "Organization",
+            "id": "http://example.org/org/123",
+            "labels": {
+              "no": "Eksempelforlaget"
+            }
+          }
+        ],
+        "role": "Creator",
+        "sequence": 0
+      }
+    ],
+    "npiSubjectHeading": "010",
+    "tags": [
+      "dokumenter",
+      "publikasjoner"
+    ],
+    "description": "En streng som beskriver innholdet i dokumentet på en annen måte enn abstrakt",
+    "abstract": "En lang streng som beskriver innholdet i dokumentet metadataene omtaler."
+  },
+  "license": {
+    "type": "License",
+    "identifier": "NTNU-CC-BY-4.0",
+    "labels": {
+      "no": "CC-BY 4.0"
+    },
+    "link": "http://example.org/license/123"
+  },
+  "fileSet": {
+    "type": "FileSet",
+    "files": [
+      {
+        "type": "File",
+        "identifier": "8c3861fa-b2dc-44af-b118-4464e042dc0b",
+        "name": "new document(1)",
+        "mimeType": "application/pdf",
+        "size": 2,
+        "license": {
+          "type": "License",
+          "identifier": "NTNU-CC-BY-4.0",
+          "labels": {
+            "no": "CC-BY 4.0"
+          },
+          "link": "http://example.org/license/123"
+        }
+      }
+    ]
+  },
+  "publisherId": "http://example.org/org/123",
+  "publisherOwnerDate": "http://example.org/org/123#eier@example.org#2020-03-30T05:51:02.487Z",
+  "@context": {
```

Comment:
Context is not embedded in the in resources in `persisted-resources`

Reply:
Looks like expanded documents/ `resources` in `persisted-resources` do have `@context`.